.github/issue_template.md

```diff
@@ -1,33 +1,70 @@
-<!--
+<!-- CLICK "Preview" FOR MORE READABLE FORMAT -->
 
 ## Prerequisites
 
 - Read the [contributing guidelines](https://github.com/sindresorhus/ava/blob/master/contributing.md).
-- Support questions are better asked in our [chat](https://gitter.im/sindresorhus/ava) or on [Stack Overflow](https://stackoverflow.com/questions/tagged/ava).
+- Support questions are better asked in one of the following locations
```

Comment:
I considered that, but you can't check them off in Preview mode, and we really do want them to delete the instructions stuff as they complete them, right? I thought people might be tempted to leave the check boxes there.